### DISCUSSION TITLE
Pairing the joy-con on emunand and sysnand

### DISCUSSION THREAD

**Discussion by Heartbeat-Heartbreak** [ndeadly/MissionControl#388] [Ideas]
>How can I use this with multiple sysNAND/emuMMC launch configs? Pairing controllers across multiple unique HOS installations requires multiple pairing databases and is essentially the same as pairing with two different consoles. The only exception being the case where you paired the controller(s) prior to making sysNAND copies. For now you will have to re-pair your controllers when switching back and forth. In the future I hope to include an option to load/store the database on the SD card to avoid this issue.

Hi, I found a simple way: https://gbatemp.net/threads/guide-pairing-the-joy-con-on-emunand-and-sysnand.562519/

**Comment by ndeadly** [comment]
Sure, that's an option to re-sync the databases if they become desynchronised, but doesn't address the root issue of desynchronisation happening in the first place, which is what the FAQ is talking about addressing. You would still need to run those scripts every time you (re)synced a controller.

Also, I'd rather not encourage users to mess around with their save files when there are better and safer ways to modify the database through homebrew. I just haven't got around to implementing anything yet.
